### PR TITLE
CVE-2016-5157 and CVE-2016-1689

### PR DESCRIPTION
**Pull request by colinthatcher**
Round one of Vulnerability History Assignment.

### CODE REVIEW

File: cves/CVE-2016-1689.yml [hunk 1]
```diff
@@ -26,14 +29,18 @@ reviews:
 - 1918073003
 bugs:
 - 606185
-repo: 
+repo: https://chromium.googlesource.com/chromium/src.git/+/e0dd9f840b3a21cd12bd3d83f5ca63302549dd21
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
```
Comment (mesternefeld) [inline cves/CVE-2016-1689.yml]: Format for this is a bit strange. It should be something like
 `- 498b074bd0db2913cf2c9458407c0d340bbcc05e:`
`<indent here>note: note goes here`
This should be the same for vccs

Comment (colinthatcher) [inline cves/CVE-2016-1689.yml]: After looking over this again, I didn't understand why that was there. Should be fixed now. Thanks!

File: cves/CVE-2016-1689.yml [hunk 2]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
```
Comment (andymeneely) [inline cves/CVE-2016-1689.yml]: Need quite a bit more on the description. See the PRs labelled "Exemplary Work" for some good examples.

File: cves/CVE-2016-1689.yml [hunk 3]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
```
Comment (andymeneely) [inline cves/CVE-2016-1689.yml]: Need that CWE

File: cves/CVE-2016-5157.yml [hunk 4]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  A Heap Buffer Overflow (out-of-bounds write) vulenrability was found in OpenJPEG, which is used
+  in PDFium in Goolge Chrome. This would allow a remote attacker to arbitrarily execute code on
```
Comment (rjahn124) [inline cves/CVE-2016-5157.yml]: From this description I don't really understand where the vulnerability was founded. Not familiar with OpenJPEG nor PDFium

File: cves/CVE-2016-5157.yml [hunk 5]
```diff
@@ -106,7 +137,7 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: I was unable to find any major events during this time.
   events:
```
Comment (rjahn124) [inline cves/CVE-2016-5157.yml]: Above it was mentioned that there were tests in CI that were talked about during the code review. Is it possible to talk about that a little bit more here?

File: cves/CVE-2016-5157.yml [hunk 6]
```diff
@@ -172,4 +208,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    The coding mistake that lead to this vulnerability was that the developer who
```
Comment (rjahn124) [inline cves/CVE-2016-5157.yml]: I actually think it's quite ironic that while trying to fix a bug, that same bug was just reproduced somewhere else on the system. 6 upvotes from me

File: cves/CVE-2016-1689.yml [hunk 7]
```diff
@@ -54,9 +61,12 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    The fix code clearly contained unit tests, and the fix included updating an
+    existing test case to better test this situation, while also adding more unit
```
Comment (philipbed) [inline cves/CVE-2016-1689.yml]: How was it updated? What did they improve?

File: cves/CVE-2016-5157.yml (hunk 6 above)
Comment (philipbed) [inline cves/CVE-2016-5157.yml]: Also, 4 upvotes from me.

File: cves/CVE-2016-1689.yml [hunk 8]
```diff
@@ -32,8 +35,12 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: e0dd9f840b3a21cd12bd3d83f5ca63302549dd21
-  :note: ''
-vccs: []
+  :note: 
+vccs:
+- :commit: aed06ff043b4d93701f2fd0c3c79f5bfd04395bb
+  :note: |
+    This commit was attempting to add functionality to generate media strams from
```
Comment (rjahn124) [inline cves/CVE-2016-1689.yml]: Not sure what a 'stram' is, should it be 'stream'?

**Pull request merged**